src/ws/test-channelresponse.c

```diff
@@ -529,9 +529,15 @@ test_resource_failure (TestResourceCase *tc,
   g_assert (cockpit_pipe_get_pid (tc->pipe, &pid));
   g_assert_cmpint (pid, >, 0);
   g_assert_cmpint (kill (pid, SIGTERM), ==, 0);
-  /* Wait until it's gone; we can't use waitpid(), it interferes with GChildWatch */
+  /* Wait until it's gone */
+#if GLIB_CHECK_VERSION(2,73,2)
```

Comment:
FTR: For a production binary that would not be good enough, as it's possible to build a binary against the old glib, and then upgrade at runtime to the new one. But for a unit test it's good enough IMHO. Maybe @allisonkarlitskaya has an idea how to make this work with both versions at runtime, but this is urgent enough to land a quick fix now.